rpcserver.go

```diff
@@ -1837,6 +1851,7 @@ func (r *rpcServer) marshalProof(ctx context.Context, p *proof.Proof,
 		IsBurn:              p.Asset.IsBurn(),
 		GenesisReveal:       genesisReveal,
 		GroupKeyReveal:      &GroupKeyReveal,
+		AltLeaves:           altLeavesBuf.Bytes(),
```

Comment:
Not sure if we should instead encode this just as a list of `taprpc.Asset` instead?

Reply:
Hmm good question...I think that would be a bit more useful, as then we can assert based on that in the future itests once everything is strung up.

Reply:
I thought about that too and decided against it, since most fields would be at some default value. And the user should never handle them as assets - this is mostly added for 3rd party tools that would need access to the leaves.